sdcm/tester.py

```diff
@@ -1225,32 +1271,52 @@ def get_cluster_k8s_gke(self):
             )
 
         if self.params.get("n_loaders"):
-            self.loaders = cluster_k8s.LoaderPodCluster(k8s_cluster=self.k8s_cluster,
-                                                        loader_cluster_config=LOADER_CLUSTER_CONFIG,
-                                                        loader_cluster_name=self.params.get("k8s_loader_cluster_name"),
-                                                        user_prefix=self.params.get("user_prefix"),
-                                                        n_nodes=self.params.get("n_loaders"),
-                                                        params=self.params,
-                                                        node_pool=loader_pool)
+            for i in range(self.k8s_cluster.tenants_number):
+                self.loaders_multitenant.append(cluster_k8s.LoaderPodCluster(
+                    k8s_cluster=self.k8s_cluster,
+                    loader_cluster_config=LOADER_CLUSTER_CONFIG,
+                    loader_cluster_name=self.params.get("k8s_loader_cluster_name") + (f"-{i + 1}" if i else ""),
+                    user_prefix=(f"{i + 1}-" if i else "") + self.params.get("user_prefix"),
+                    n_nodes=self.params.get("k8s_n_loader_pods_per_cluster") or self.params.get("n_loaders"),
+                    params=self.params,
+                    node_pool=loader_pool,
+                    add_nodes=False,
+                ))
+            self.loaders = self.loaders_multitenant[0]
+
+        # NOTE: wait for Scylla and loader clusters nodes creation
```

Comment:
what's changed that we need to wait for them to finish now ? how did worked before ? we didn't wait ?

Reply:
o.k